report/renderer/report-utils.js

```diff
@@ -142,6 +143,27 @@ class ReportUtils {
     }
   }
 
+  /**
+   * Returns a comparator created from the supplied list of keys
+   * @param {Array<string>} sortedBy
+   * @return {((a: LH.Audit.Details.TableItem, b: LH.Audit.Details.TableItem) => number)}
+   */
+  static getTableItemSortComparator(sortedBy) {
```

Comment:
This has nothing to do with table items. It's pretty generic. Maybe `getComparatorForKeys`/`getComparatorForProperties`?

Reply:
So one thought I have is — we might want to show "1st Party" group as the topmost group, regardless of its values, to acknowledge the actionability aspect. That would make it very specific to a comparator for the table/group rows. Thoughts?